ci/jenkins/test-vmc.sh

```diff
@@ -185,10 +235,138 @@ function release_static_ip() {
     echo Released IP
 }
 
-function setup_cluster() {
+function setup_aws_sg() {
+    # New control plane security group
+    set +e
+    clusterawsadm bootstrap iam create-cloudformation-stack --region $AWS_REGION
+    set +x
+    export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile --region $AWS_REGION)
+    set -x
+    export GOPROXY=direct
+    clusterctl init --infrastructure aws
+    unset GOPROXY
+    
+    aws ec2 create-security-group --description capi-controlplane --group-name capi-controlplane --vpc-id $AWS_VPC_ID --region $AWS_REGION
+    export CONTROLPLANE_SGID=$(aws ec2 describe-security-groups --region $AWS_REGION --group-names capi-controlplane --query 'SecurityGroups[0].GroupId' --output text)
+    # All allow from subnet
```

Comment:
Do you mean to allow all from subnet?
maybe:
```suggestion
    # Allow all from subnet
```